soco/core.py

```diff
@@ -1287,6 +1288,35 @@ def add_to_queue(self, queueable_item):
         qnumber = response['FirstTrackNumberEnqueued']
         return int(qnumber)
 
+    def add_multiple_to_queue(self, item_list, container=None):
```

Comment:
suggest to make the argument name "items", not item list, since the only requirement of the code is that it is a sequence of items.

This applies also to the entire docstring, where all references to this argument should list it as a sequence of items, with the only exceptions being the argument type, where it makes sense to list it as a list.